prime-router/src/main/kotlin/tokens/OktaAuthentication.kt

```diff
@@ -102,6 +100,14 @@ class OktaAuthentication(private val minimumLevel: PrincipalLevel = PrincipalLev
         }
     }
 
+    fun decodeJwt(accessToken: String): Jwt {
+        val jwtVerifier = JwtVerifiers.accessTokenVerifierBuilder()
+            .setIssuer("https://$issuerBaseUrl/oauth2/default")
+            .build()
+        // Perform authentication.  Throws exception if authentication fails.
+        return jwtVerifier.decode(accessToken)
```

Comment:
this is where the meat of the Okta code happens.
 this method reaches out to the Okta verifier (https://hhs-prime.oktapreview.com/oauth2/default) and returns if the token is verified or not.
separating this out makes it so that we do not have to go all the way to Okta to get a good token, and we can now mock the decoding of the token and focus on the result of the claims